appsdk/sdk_test.go

```diff
@@ -456,8 +446,8 @@ func TestInitializeClientsJustData(t *testing.T) {
 }
 
 func TestValidateVersionMatch(t *testing.T) {
-	clients := make(map[string]common.ClientInfo)
-	clients[common.CoreDataClientName] = common.ClientInfo{
+	clientMap := make(map[string]common.ClientInfo)
```

Comment:
Don't use type in the variable name. Leave as clients

Reply:
`clients` collides with a package name and generates a linter warning.

Reply:
:-( Ok, how about `coreClients`

Reply:
Applies to my similar comments.